### PR TITLE
Update Link Styles

### PR DESCRIPTION
**Pull request by jehoshua02**
Bemify link styles to elminate attribute selectors.
Update link styles with all link states and different variables for state colors.

### CODE REVIEW

File: dist/scss/_link.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  color: $link-color;
+  font-family: inherit;
+  text-decoration: none;
+
+  &:visited {
+    color: $link-color-visited;
+  }
+
+  &:hover,
+  &--hover {
```
Comment (janiv) [inline dist/scss/_link.scss]: Remove &--hover. Can you sneak in a &:focus real quick too?

File: dist/scss/_link.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    color: $link-color-hover;
+    text-decoration: underline;
+    cursor: pointer;
+  }
+
+  &:active,
+  &--active {
```
Comment (janiv) [inline dist/scss/_link.scss]: Remove &--active

File: src/scss/_link.scss [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  }
+
+  &--forward:after {
+    content: " \00bb"; /* &raquo; */
+  }
+
+  &--backward:before {
+    content: "\00ab  "; /* &laquo; */
+  }
+}
```
Comment (janiv) [inline src/scss/_link.scss]: See all my comments that I made on /dist and meant to put on /src

**Comment by jehoshua02** [comment]
@janiv Updates according to feedback. There was also a merge conflict. So I did a `git rebase -i master`. And it looked like a conflict in `dist` and `docs`. So I did `gulp clean; gulp build`.

**Comment by janiv** [comment]
Leaving it up to you tag and release.

**Pull request merged**